### DISCUSSION TITLE
Use with HiFiBerry OS

### DISCUSSION THREAD

**Discussion by Primo83** [music-assistant/support#383] [Q&A] [answered]
It would be nice if there was support for HiFiBerry DACs with hifiberryos for hight quality music. https://github.com/hifiberry/hifiberry-os

https://www.hifiberry.com/

**Comment by marcelveldt** [comment, marked as answer]
There already is! I have recently contributed the "slimproto" integration to HA core.

1) Enable squeezelite on HifiBerry OS
2) Add "slimproto" (squeezebox players) integration to Home Assistant.
3) Your HIFIBerry will be automatically added to Home Assistant as mediaplayer
4) Add that mediaplayer to Music Assistant
5) voila, Hi res lossless audio streaming without any additional media servers

**Reply by mironfs** [reply]
hi, i added squeezelite as provider directly inside music assistant without downloading slimproto. is there any negative using it like this? thanks

**Reply by OzGav** [reply]
Sorry this is from 3 years ago and is no longer relevant. What you done is correct.
